docs/source/using/experimental-features.adoc

```diff
@@ -87,3 +87,29 @@ PS> minishift.exe start `
 Be sure to specify a valid gateway and nameserver.
 Failing to do so will result in connectivity issues.
 ====
+
+
+[[set-fixed-ip]]
+== Set Fixed IP Address
+
+[NOTE]
+====
+Currently not supported on KVM as the driver plugin relies on the DHCP offer to determine the IP address.
+====
+
+While the VM gets assigned an IP address using DHCP, most hypervisors do not handle extending the lease time of the assignment.
+This can cause issues on restarts which assigns a new IP address to the instance. While the VM will start, it will conflict with the security certificates that have been generated before and the deployment will fail.
+
```

Comment:
Please address the previous comment for this section. 
https://github.com/minishift/minishift/pull/1849#discussion_r176375812